server/models/ticket.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from 'mongoose';
+
+const TicketSchema = new mongoose.Schema({
+  groupId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+  },
+  metadata: mongoose.Schema.Types.Mixed
+});
+
+
+/**
+ * Statics
+ */
+TicketSchema.statics = {
```

Comment:
We are going to need to list tickets in the UI, so I'm confident this will be used.